swift/Workflow/Sources/Worker.swift

```diff
@@ -44,3 +44,48 @@ extension Worker where Self: Equatable {
     }
 
 }
+
+extension Worker {
+
+    public func asAnyWorkflow() -> AnyWorkflow<Void, Output> {
+        return WorkerWorkflow(worker: self).asAnyWorkflow()
+    }
+
+}
+
+
+import Foundation
+
+
+fileprivate struct WorkerWorkflow<WorkerType: Worker>: Workflow {
+
+    var worker: WorkerType
+
+    typealias Output = WorkerType.Output
+
+    typealias State = UUID
+
+    func makeInitialState() -> State {
+        return UUID()
+    }
+
+    func workflowDidChange(from previousWorkflow: WorkerWorkflow<WorkerType>, state: inout UUID) {
+        if !worker.isEquivalent(to: previousWorkflow.worker) {
+            state = UUID()
+        }
+    }
+
+    typealias Rendering = Void
+
+    func render(state: State, context: RenderContext<WorkerWorkflow>) -> Rendering {
+        context.runSideEffect(key: state) { sink, lifetime in
```

Comment:
I know you ended up removing this sink, but passing an explicit sink here would let us keep backpressure, as I raised in https://github.com/square/workflow/issues/1021#issuecomment-598976131.